hypothesis-python/src/hypothesis/internal/conjecture/providers.py

```diff
@@ -827,3 +827,30 @@ def draw_bytes(
     ) -> bytes:
         values = self._draw_collection(min_size, max_size, alphabet_size=2**8)
         return bytes(values)
+
+
+class URandom(Random):
```

Comment:
We can either leave registering this to the antithesis python sdk, or register it ourselves. I'm leaning towards the former, since I don't expect `backend="urandom"` to be useful outside of antithesis.

Reply:
IMO we should register a `hypothesis-urandom` backend, document the current list of known backends (inc crosshair) as a rst definition-list in the alternative-backends doc section, and mention in the changelog that you can enable it using the `backend="hypothesis-urandom"` setting (which is valuable for antithesis users and a pointless slowdown for anyone else).

I _don't_ expect it to be useful to non-Antithesis-users, but the integration has very few downsides and I like advertising the possibilities that alternative backends can enable.  We might also want to tone down the instability warnings a bit...